aio/content/examples/animations/src/app/app.component.ts

```diff
@@ -22,8 +22,10 @@ import { slideInAnimation } from './animations';
   animations: [
 // #enddocregion decorator
     slideInAnimation
+// #enddocregion toggle-app-animations, define
 // #docregion decorator
```

Comment:
Nit: (Although either way works) I think the interleaved docregions would be a little easier to grok if these too lines were swapped:

```suggestion
// #docregion decorator
// #enddocregion toggle-app-animations, define
```

Reply:
yeah, sorry I must have gotten my wires crossed, this is of course much much better :sweat: